test/unit/EOFeedManager.t.sol

```diff
@@ -30,7 +30,7 @@ contract EOFeedManagerTest is Test, Utils {
     address private notOwner = makeAddr("notOwner");
     uint16 private feedId = 1;
     uint256 private rate = 1_000_000;
-    uint256 private timestamp = 9_999_999_999;
+    uint256 private timestamp = 1_801_180_800;
```

Comment:
should we add tests to check ?
1. on chain storage with ms while input is seconds 
2. on chain storage with seconds and input is ms